.infrastructure/bootstrap.sh

```
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -euo pipefail
```

Good: `set -euo pipefail` is present which ensures the script fails fast on errors as requested in the checklist. This addresses the previous suggestion to add fail-fast safety.